test/ssrf-parity.test.ts

```
@@ -0,0 +1,337 @@
+}
+
+interface NetworkHelper {
+  getNetworkEntries(): { ipv4: NetworkEntry[]; ipv6: NetworkEntry[]; names: NameEntry[] };
+  isPrivateHostname(hostname: string): boolean;
+}
+
+function loadHelper(modulePath: string, buildHint: string): NetworkHelper {
+  try {
+    return require(modulePath) as NetworkHelper;
+  } catch (error) {
+    const err = error as NodeJS.ErrnoException;
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)